wdl/transforms/new-base/src/main/scala/wdl/transforms/base/linking/expression/values/EngineFunctionEvaluators.scala

```diff
@@ -472,8 +476,10 @@ object EngineFunctionEvaluators {
                                ioFunctionSet: IoFunctionSet,
                                forCommandInstantiationOptions: Option[ForCommandInstantiationOptions])
                               (implicit expressionValueEvaluator: ValueEvaluator[ExpressionElement]): ErrorOr[EvaluatedValue[WomInteger]] = {
-      processValidatedSingleValue[WomFloat, WomInteger](a.param.evaluateValue(inputs, ioFunctionSet, forCommandInstantiationOptions)) { float =>
-        EvaluatedValue(WomInteger(math.round(float.value).toInt), Seq.empty).validNel
+      import BigDecimal.RoundingMode.HALF_UP
+      def round(a: BigDecimal): BigDecimal = a.setScale(0, HALF_UP)
+      processValidatedSingleValue[WomFloat, WomInteger](a.param.evaluateValue(inputs, ioFunctionSet, forCommandInstantiationOptions)) { bigDecimal =>
```

Comment:
This line is 143 character long, consider splitting it into two lines.

Reply:
Done